fireant/slicer/widgets/reacttable.py

```diff
@@ -163,6 +164,41 @@ def map_display_values(df, dimensions):
 
         return dimension_display_values
 
+    @staticmethod
+    def map_hyperlink_templates(df, dimensions):
+        """
+        WRITEME
+
+        :param df:
+        :param dimensions:
+        :return:
+        """
+        hyperlink_templates = {}
+        pattern = re.compile(r'{[^{}]+}')
+
+        for dimension in dimensions:
+            hyperlink_template = dimension.hyperlink_template
+            if hyperlink_template is None:
+                continue
+
+            arguments = [format_dimension_key(argument[1:-1])
+                         for argument in pattern.findall(hyperlink_template)]
+            missing_dimensions = set(arguments) & set(df.index.names)
+            if not missing_dimensions:
```

Comment:
Minor but I think this could have a clearer name. Instead of `missing_dimensions` this could be `dimensions_to_hyperlink` or something.

Reply:
Could rename it to `unavailable_hyperlink_parameters`. This is a set of the dimensions required to fill in the hyperlink template that are not in the result data set.